jinglify/Game/Game.swift

```diff
@@ -54,6 +57,21 @@ class Game {
         }
     }
 
+    func pauseGame(){
+        isPaused = true
+        statusText = "Paused"
+        if isJinglePlaying {
+            self.player.pauseJingle()
+        }
```

Comment:
It seems that invalidating and restarting `gameTimer` (as well as other ongoing timers) will be more robust way to pause a game. Also we need to respond to app lifecycle events and [UIApplicatonDelegate doc](https://developer.apple.com/reference/uikit/uiapplicationdelegate/1622950-applicationwillresignactive)  recommends to invalidate timers also.